public/index.html

```diff
@@ -433,27 +433,25 @@ <h5 class="modal-title" id="exampleModalLabel">Modal title</h5>
 		<div class="container-fluid">
 			<div class="well row attributesWrapper" style="margin-top: 1%;">
 				<div class="col-md-3 wsdlIcon" >
-					<img src="/images/odu-sig-noidea-fullcolor.png" alt="ODULogo">
 					<img src="/images/glider-trans.png" alt="ODUWDL">
 				</div>
 				<div class="col-md-6" >
-					<p> Created by the <a href ="https://ws-dl.cs.odu.edu/">Web Science and Digital Libraries Research Group</a> at <a href ="https://cs.odu.edu/">Old Dominion University</a>. </p>
-					<p>Funded by
-						<a href="https://securegrants.neh.gov/publicquery/main.aspx?f=1&gn=HAA-256368-17"> NEH/IMLS Digital Humanities Advancement Grant HAA-256368-17</a>
-					</p>
+					<p style="padding-top: 10px;"> Created by the <a href ="https://ws-dl.cs.odu.edu/">Web Science and Digital Libraries Research Group <a href="https://twitter.com/WebSciDL?ref_src=twsrc%5Etfw">(@WebSciDL)</a></a> at <a href ="https://cs.odu.edu/">Old Dominion University</a>. </p>
 					<p>With our partners
 						<a href="https://www.frick.org/research/library"> Frick Art Reference Library</a>,
 						<a href="http://www.nyarc.org/">New York Art Resources Consortium</a> and
 						<a href="http://library.columbia.edu/">Columbia University Libraries</a>
 					</p>
+					<p>Funded by
+						<a href="https://securegrants.neh.gov/publicquery/main.aspx?f=1&gn=HAA-256368-17"> NEH/IMLS Digital Humanities Advancement Grant HAA-256368-17</a>
+						<a href="https://ws-dl.blogspot.com/2017/10/2017-10-16-visualizing-webpage-changes.html"> (project intro blog post)</a>
+					</p>
 					<br>
-					<a href= "http://ws-dl.blogspot.com/2017/10/2017-10-16-visualizing-webpage-changes.html">Full Description of the Project</a>
-					<br>
-					<a href= "https://github.com/oduwsdl/tmvis">GitHub Repository</a>
+					<p>For source code, issue reports, and more documentation see the project's <a href= "https://github.com/oduwsdl/tmvis">GitHub repository</a></p>
 
 				</div>
 				<div class="col-md-3" >
-					<a href="https://twitter.com/WebSciDL?ref_src=twsrc%5Etfw" class="twitter-follow-button" data-size="large" data-show-count="false">Follow @WebSciDL</a><script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+					<img style="padding-top: 20px;" src="/images/odu-sig-noidea-fullcolor.png" width="250px" alt="ODULogo">

```

Comment:
Alt text is for screen readers, make it something that they can meaningfully read (hence I added a space).

```suggestion
					<img style="padding-top: 20px;" src="/images/odu-sig-noidea-fullcolor.png" width="250px" alt="ODU Logo">
```